yarn-project/txe/src/oracle/txe_oracle.ts

```diff
@@ -209,27 +201,6 @@ export class TXE implements TypedOracle {
     return inputs;
   }
 
-  async avmOpcodeNullifierExists(innerNullifier: Fr, targetAddress: AztecAddress): Promise<boolean> {
-    const nullifier = siloNullifier(targetAddress, innerNullifier!);
-    const db = await this.trees.getLatest();
-    const index = await db.findLeafIndex(MerkleTreeId.NULLIFIER_TREE, nullifier.toBuffer());
-    return index !== undefined;
-  }
-
-  async avmOpcodeEmitNullifier(nullifier: Fr) {
-    const db = await this.trees.getLatest();
-    const siloedNullifier = siloNullifier(this.contractAddress, nullifier);
-    await db.batchInsert(MerkleTreeId.NULLIFIER_TREE, [siloedNullifier.toBuffer()], NULLIFIER_SUBTREE_HEIGHT);
-    return Promise.resolve();
-  }
-
-  async avmOpcodeEmitNoteHash(noteHash: Fr) {
```

Comment:
There's too many changes in this file, were these functions simply moved around, or did you actually change them? It's quite difficult to parse what happened here given the diff.

Reply:
Simply moved around so all AVM oracles are together. As discussed with @fcarreiro, they should be mostly unused, since now we are always doing nested calls through the simulator. Unfortunately, that's not true if you want to test a  public state var in isolation, so I wanted to keep them around...just a little bit more organized.